avm/ptn/app/container-job-toolkit/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,129 @@
+targetScope = 'subscription'
+
+metadata name = 'Pass in existing resources'
+metadata description = 'This instance deploys the module with existing resources.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-app-containerjob-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'acjmax'
```

Comment:
If renaming the folder please also rename the serviceshort accordingly

Reply:
Not needed (see above comment)